src/screens/ProfileScreen/elements/AnonymousTab.js

```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 import {StyleSheet, Text, View} from 'react-native';
 
 import AnonymousIcon from '../../../assets/icon/AnonymousIcon';
-import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors` to `COLORS` is a good one as it follows the convention of using uppercase for constant values. This makes the code more readable and maintainable.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->